### DISCUSSION TITLE
GNOME-like icon

### DISCUSSION THREAD

**Discussion by oiimrosabel** [andyholmes/valent#584] [General]
I tried to make an Adwaita-like icon, respecting all the GNOME HIG. What do you think ?

![image](https://github.com/andyholmes/valent/assets/42437280/b30370af-7143-4b3b-9136-e9dbf04f13a7)

**Comment by andyholmes** [comment]
Thanks, it kind of looks like the old one though?

![image](https://github.com/andyholmes/valent/assets/1265208/d67dacee-242e-4976-ad94-18ee8cc5819c)

Maybe you could collaborate with @diegopvlk if there's some improvements you want to make?
